conf/defaults.ini

```diff
@@ -473,6 +473,9 @@ auto_assign_org_role = Viewer
 # Require email validation before sign up completes
 verify_email_enabled = false
 
+# Redirect to default OrgId after login
+login_default_org_id = -1
```

Comment:
Add this to sample ini

Reply:
I think you can use empty value instead if -1 as default

Reply:
Thanks, done!